pkg/framework/aries/framework.go

```diff
@@ -46,7 +46,8 @@ type Aries struct {
 // Option configures the framework.
 type Option func(opts *Aries) error
 
-// New initializes the Aries framework based on the set of options provided.
+// New initializes the Aries framework based on the set of options provided. This function returns a framework
+// which can be used to manage Aries clients by getting the framework context.
```

Comment:
Should also create examples. See DID exchange godoc.

Reply:
[Added example](https://github.com/hyperledger/aries-framework-go/compare/1a0fabad4135a7495c46315752ad06c326249a89..6d8821c860b3f4c2f12b4e5048eacf470b90bcd2#diff-6b0dce919a202a9ca9791c186a715b5cR17).